Source/Core/Core/HW/SystemTimers.cpp

```diff
@@ -140,8 +135,13 @@ static void SICallback(u64 userdata, int cyclesLate)
 
 static void CPCallback(u64 userdata, int cyclesLate)
 {
-	CommandProcessor::Update();
-	CoreTiming::ScheduleEvent(CP_PERIOD - cyclesLate, et_CP);
+	static u64 last_call = CoreTiming::GetTicks();
```

Comment:
Are static variables a wise thing to use with regards to running two emulation sessions without restarting the program?